README.txt

```diff
@@ -32,9 +32,70 @@ Portability
 
 Ease of use
     The code follows the `Python Packaging User Guide
-    <http://python-packaging-user-guide.readthedocs.org/en/latest/>`_. This will
-    make installing the ``ga4gh`` reference code very easy across a range of
-    operating systems.
+    <http://python-packaging-user-guide.readthedocs.org/en/latest/>`_. 
+    Specifically, pip is used to handle python package dependencies (see below 
+    for details). This allows for easy installation of the ``ga4gh`` reference code 
+    across a range of operating systems.
+
+
+*******************************
+Installing the reference server
+*******************************
+
+Prerequisites:
+
+- Python 2.7 or higher,
+- Berkeley DB together with include and lib files (version 48 or higher),
+- Virtualenv (or another python sandboxing tool) is highly recommended.
+
+General installation procedure:
+
+- Install Berkeley DB (version 48 or higher) using your system's preferred 
+  package manager, see the `wormtable help page 
+  <https://pypi.python.org/pypi/wormtable>`_ for platform-specific details.
+  In one developer's experience, installing db48 was preferable to db53 
+  on MacOS X via MacPorts. For example (on MacOS X with MacPorts):
+
+  $ sudo port install db48
+
+- Set the LDPATH and CPATH environment variables to include the lib 
+  and include directories for the Berkeley DB install of your choice. 
+  The wormtable help page provides platform-specific examples.
+  For example (again, on MacOS X with Berkeley DB installed as above,
+  your specific paths to include and lib directories may differ):
+
+  $ export CFLAGS=-I/opt/local/include/db48/  
+  $ export LDFLAGS=-L/opt/local/lib/db48/
+
+- Create a python sandbox directory using virtualenv, preferably 
+  NOT inside the ga4gh server directory. For an good introduction
+  to using virtualenv, see the `Python Guide page
+  <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_.
+  On some systems, you may need to specify the --no-site-packages
+  option to ensure a clean dependency install. For example, to 
+  create a sandbox named ``testenv``:
```

Comment:
This is RST, so we need two colons for these example sections to get the rendering right. i.e. " create a sandbox named ``testenv``::"